static/js/common/upload-addon.js

```diff
@@ -76,7 +76,7 @@
         }),
         ui_details = $('<div>', {
           class: 'upload-details',
-          text: gettext('Your add-on should end with .zip, .xpi, .crx or .xml'),
+          text: gettext('Your add-on should end with .zip, .xpi, or .crx'),
```

Comment:
Nit: previous version did not use an Oxford comma. I'm guessing this is a typo and should be removed here as well.

Reply:
Good catch, fixed in 0ec5a367758bd1b2d8d6280786e297f0e9964b04